website/templates/project/edit_draft_registration.mako

```diff
@@ -48,7 +48,7 @@
 
                 <div class="row" style="margin-bottom: 10px;">
                   <span>
-                    Last saved: <span data-bind="text: lastSaveTime"></span>
+                    Last auto-saved: <span data-bind="text: lastSaveTime"></span>
```

Comment:
Cool, thank you. When you do a review with Nici and/or Sara, please mention this explicitly and get their feedback as well, just in case.